### DISCUSSION TITLE
Is there any way to automate a js prompt with two inputs?

### DISCUSSION THREAD

**Discussion by Jegomezre** [webdriverio/webdriverio#12451] [Q&A]
I want to automate a login flow that prompts what is shown in the image attached. Is there any way to automate the input of both values?

![Snag_b5aae4a](https://github.com/webdriverio/webdriverio/assets/92931023/4553b3db-c38f-42d2-ad0a-b26c8df1a443)

**Comment by christian-bromann** [comment]
Yes, WebdriverIO is working on making this automation process easier.
